endpoints/openrtb2/auction_test.go

```diff
@@ -495,7 +495,7 @@ func doBadAliasRequest(t *testing.T, filename string, expectMsg string) {
 	bidderInfos := getBidderInfos(nil, openrtb_ext.CoreBidderNames())
 
 	bidderMap := exchange.GetActiveBidders(bidderInfos)
-	disabledBidders := exchange.GetDisabledBiddersErrorMessages(bidderInfos)
+	disabledBidders := exchange.NewDisabledBidders().GetWarningMessages(bidderInfos)
```

Comment:
I was on the fence between:

`exchange.NewDisabledBidders().GetWarningMessages(bidderInfos)`

and

`exchange.NewDisabledBidders(bidderInfos).GetWarningMessages()`

Which do you prefer?

Reply:
Yes to simplified version!